databases/foundationdb/Portfile

```diff
@@ -5,28 +5,44 @@ PortGroup           github 1.0
 PortGroup           cmake 1.1
 PortGroup           compiler_blacklist_versions 1.0
 
-github.setup        apple foundationdb 6.3.12
-revision            1
+github.setup        apple foundationdb 6.3.24
+revision            0
 
 categories          databases
 license             Apache-2
 maintainers         {me.com:spam_brian @brianwells} openmaintainer
 platforms           darwin
-description         FoundationDB is a distributed database providing ACID transactions.
-long_description    FoundationDB is a scalable, fault-tolerant, ordered \
-                    key-value store with full ACID transactions.  This port \
-                    will install version ${version} of the FoundationDB \
-                    binaries suitable for development on macOS.
+description         FoundationDB gives you the power of ACID transactions in a distributed database.
+long_description    FoundationDB is a distributed database designed to handle large volumes of structured data \
+                    across clusters of commodity servers. It organizes data as an ordered key-value store and \
+                    employs ACID transactions for all operating systems. It is especially well-suited for read/write workloads \
+                    but also has excellent performance for write-intensive workloads. Users interact with the \
+                    database using API language binding. FoundationDB is Open source, is Ready for production, \
+                    provides Industry-leading performance, is Fault tolerant and is Multi-model. This Port provides \
+                    Latest FoundationDB Release to macOS with full support for all these features.
 
 homepage            https://www.foundationdb.org/
 
-checksums           rmd160  2bf9a258d5db140d232fb6355fc810010824c014 \
-                    sha256  034f69d4ddf7faccd7cae2a6438d64643ac98a3a9c7b3e6ed2a03105c480df95 \
-                    size    9694306
+checksums           rmd160  470b145f9974cc0898a176425d72f9bc43704248 \
+                    sha256  e5ab9925d1a1f2f3ceb48ce4d3c5211176aedf3f04df7d3945b578f7646b9fb2 \
+                    size    10654924
 
 # Required python version
-set py_ver          3.9
+set py_ver          3.10
 set py_ver_nodot    [string map {. {}} ${py_ver}]
+set jdk_ver         8
+set ruby_ver        3.1
+set ruby_ver_nodot  [string map {. {}} ${ruby_ver}]
+default_variants    +python${py_ver_nodot}
+
+variant openjdk${jdk_ver} description {add java binding support with openjdk meta port} {
```

Comment:
Inside the braces for each variant is where you add the dependencies and build arguments that would enable that variant in the build process:

```
variant openjdk${jdk_ver} description {add java binding support with openjdk meta port} {
    depends_build-append port:openjdk-...
    build.args-append -DENABLE_JAVA.... # whatever build args specifically enable Java
}
```